### PR TITLE
Add the tower widths in eta to geometry.

### PR DESCRIPTION
**Pull request by benkrikler**
This is useful for some PUS schemes which account for the different Eta widths.

I might be duplicating code here, since I'm guessing something like this is available in cmssw already.  If so, could someone point me to it and I'll close this?

### CODE REVIEW

**Comment by kreczko** [comment]
The PR has failed the tests (pep8), but the info does not appear here:
https://travis-ci.org/cms-l1t-offline/cms-l1t-analysis

File: cmsl1t/geometry.py [hunk 1]
```diff
@@ -25,3 +25,19 @@ def is_in_region(region, eta, regions=eta_regions):
         raise KeyError(msg)
 
     return regions[region](eta)
+
+
+__etaSizes_21onwards=[0.09 , 0.1   , 0.113 , 0.129 , 0.15, 0.178 ,
```
Comment (benkrikler) [inline cmsl1t/geometry.py]: Good thought, I hadn't tried looking for one.

**Comment by kreczko** [comment]
If you run `make flake8` or `make test` locally, you should be able to check things before you upload:
```
cmsl1t/geometry.py:31:9: E126 continuation line over-indented for hanging indent
cmsl1t/geometry.py:33:9: E123 closing bracket does not match indentation of opening bracket's line
cmsl1t/geometry.py:44:47: E226 missing whitespace around arithmetic operator
```

**Comment by benkrikler** [comment]
I ran flake8 myself before I uploaded it, but it didn't flag the issue with indentations.  I'm not sure if my version if more relaxed than full PEP-8 compliance, or if the version you have in travis is stricter.

**Pull request merged**